### PR TITLE
Updating content across the site

### PR DESCRIPTION
**Pull request by rachel-dtr**
## Description

Continuing to clean up some content across the site.

### CODE REVIEW

File: atst/utils/flash.py [hunk 1]
```diff
@@ -162,7 +160,7 @@
         "title_template": translate("flash.success"),
         "message_template": """
             {{ "flash.application.deleted" | translate({"application_name": application_name}) }}
-            <a href="#">Undo</a>.
+            <a href="#">translate("common.undo")</a>
```
Comment (drummond-dds) [inline atst/utils/flash.py]: I think we want to call this as 

```
<a href="#">{{ "common.undo" | translate }}</a>
```

File: atst/forms/portfolio_member.py [hunk 2]
```diff
@@ -14,29 +14,29 @@ class PermissionsForm(BaseForm):
     perms_app_mgmt = SelectField(
         translate("forms.new_member.app_mgmt"),
         choices=[
-            (PermissionSets.VIEW_PORTFOLIO_APPLICATION_MANAGEMENT, "View only"),
-            (PermissionSets.EDIT_PORTFOLIO_APPLICATION_MANAGEMENT, "Edit access"),
+            (PermissionSets.VIEW_PORTFOLIO_APPLICATION_MANAGEMENT, "View"),
```
Comment (drummond-dds) [inline atst/forms/portfolio_member.py]: These should reference the `translations.yaml`. I can help you get this sorted on Monday if you need help finding the correct translation keys.

**Comment by drummond-dds** [comment]
Specs are failing because it looks like we have hard coded strings in the test files. We can get an engineer to pair with you on this to get them passing again.

**Comment by montana-mil** [comment]
Ready for another look!

**Pull request merged**